src/core/command.cpp

```diff
@@ -317,15 +322,16 @@ bool Command::set_attribute(JsonObject output, const char * cmd, const char * at
 
     // attribute isn't found
     // it could be a value command, but the value doesn't exist?
-    if (strcmp(attribute, "value") == 0) {
+    if (strncmp(attribute, "value", 5) == 0) {
```

Comment:
strcmp compares to the terminating zero, this case max to 6th char in "value", with strncmp it matches any name that starts with "value".

Reply:
good spot, I'll change it.